apis/r/tools/get_tarball.R

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env Rscript
+
+## version pinning info
+tiledb_core_version <- "2.15.2"
```

Comment:
to minimize keystroking on future dependency-upgrade PRs, @gspowley wrote a very nice `scripts/update-tiledb-version.py`:

it looks like that script might not update this current script as-is -- ?

updating that script to handle this file might be done on this PR, or on a follow-up PR

Reply:
I wondered about that too but I thought the above was better because we only to change _once_ for version and sha1 rather than three times (with largely redundant long URLs):

https://github.com/single-cell-data/TileDB-SOMA/blob/326d5eac52ba6fd65ea2fda62faa8e7cf5a6c1a7/apis/r/tools/get_tarballs.R#L8-L19

Easy enough to revert if that is what we prefer.  In tiledb-r I actually update two-line text 'key: value' text file that is then read (via a [[DCF]()](https://www.debian.org/doc/debian-policy/ch-controlfields.html) reader R has in base so no need to worry about jaml or json...):

https://github.com/TileDB-Inc/TileDB-R/blob/870f9343bd4fab056e8e46ee83b23acfb47b423d/tools/tiledbVersion.txt#L1-L2